src/emulator/functionsEmulator.ts

```diff
@@ -699,17 +699,18 @@ export class FunctionsEmulator implements EmulatorInstance {
     }
 
     const idToken = match[1];
-    const tokenParts = idToken.split(".");
-
-    if (tokenParts.length < 2) {
-      return;
-    }
+    logger.debug(`ID Token: ${idToken}`);
```

Comment:
ID tokens expire in an hour so I am not worried about logging this (and it is valuable) but if anyone disagrees I will remove.